src/neuroglancer/perspective_view/panel.ts

```diff
@@ -229,6 +231,56 @@ export class PerspectivePanel extends RenderedDataPanel {
       });
     });
 
+    registerActionListener(element, 'select', (e: ActionEvent<MouseEvent>) => {
```

Comment:
Can we avoid the code duplication here, but factoring out common logic?

I'm actually not that keen on overriding the meaning of the select command here.  Additionally, startRelativeMouseDrag was really intended to be used only on mousedown events.  It happens to work, but it leaves room for weird behavior, like both moving an annotation and placing another annotation at the same time.

I think it would be better to revert to the original behavior of using alt+mousedown0 for now.  I think it would be better that alt+mousedown0 starts moving, and alt+mousedown0 again stops moving, and the two clicks need not even be in the same panel.  However, implementing that would make this change more complicated, so it could wait until later.

Reply:
I think I could use a little more guidance to address this.  I've switched to the alt+mouse0down functionality, so you have are mouseDragging again, rather than picking up and putting down. 

However, my attempts of reducing code duplication have all been rather ugly.  There are certaintly many duplicate lines of code, but with subtle different lines interspersed, and I'm having trouble finding the right abstraction to remove the duplicate part replication without having a lot of ugly options.